client/src/proxies/proxy.js

```diff
@@ -40,8 +40,10 @@ export default class BaseProxy {
     const method = 'POST';
 
     try {
-      const response = await axios.post(requestUrl, data);
-      return response.data;
+      const response = await axios.post(requestUrl, data, {
+        withCredentials: true,
```

Comment:
the credentials include should pass as optional param in ALL requests , 
you have also authenticated request also on get/delete/put method APIs, 
not only on post APIs